File: problemas/1000/1048.cs
using System;

namespace uri1048
{
    internal static class Program
    {
        private static void Main()
        {
            double salario = double.Parse(Console.ReadLine());

            double indice = salario <= 400.00
                ? 0.15
                : salario <= 800.00
                    ? 0.12
                    : salario <= 1200.00
                        ? 0.1
                        : salario <= 2000.00
                            ? 0.07
                            : 0.04;

            double reajuste = salario * indice;

            Console.WriteLine($"Novo salario: {salario + reajuste:F2}");
            Console.WriteLine($"Reajuste ganho: {reajuste:F2}");
            Console.WriteLine($"Em percentual: {indice * 100:F0} %");
        }
    }
}

File: problemas/1000/1048.php
<?php

$salario = readline();

$indice = $salario <= 400.00
    ? 0.15
    : ($salario <= 800.00
        ? 0.12
        : ($salario <= 1200.00
            ? 0.1
            : ($salario <= 2000.00
                ? 0.07
                : 0.04)));

$reajuste = $salario * $indice;

echo 'Novo salario: ' . number_format($salario + $reajuste, 2, '.', '') . "\n";
echo 'Reajuste ganho: ' . number_format($reajuste, 2, '.', '') . "\n";
echo 'Em percentual: ' . number_format($indice * 100, 0) . " %\n";